Assets/Plugins/Mixpanel/Mixpanel.cs

```diff
@@ -25,7 +25,7 @@ public static class Mixpanel
 
 	private const string API_URL_FORMAT = "https://api.mixpanel.com/track/?data={0}";
 	private static MonoBehaviour _coroutineObject;
-	private static bool sendInEditor = false;
+    private static bool sendInEditor = false;
 
 	public static void ToggleSendInEditor(bool isEnabled)
```

Comment:
@gracebarsny this change is done and the rename. 

However, I disagree about the inline array declaration. Don't believe we ever determined a convention and would prefer how it's currently written